### ISSUE TITLE
Nginx returns 404 instead of 50x when backend service is down

### ISSUE THREAD

**Issue by znerol** [bigbluebutton/bigbluebutton#13025]
**Describe the bug**

When some backend service is down (greenlight, one of the html5 frontend/backend services), Nginx will return a _404 Not Found_ instead of a _503 Service Unavailable_.

**To Reproduce**

Freshly installed BigBlueButton with Greenlight running in a docker container.

1. Restart the server
2. Repeatedly try to load greenlight in the browser (or using `curl`).

**Expected behavior**

Nginx should return the correct status 503 whenever a backend service is unavailable.

**Actual behavior**

Nginx returns 404 in the timespan between startup of Nginx and startup of the Greenlight docker container.

Also the `/var/log/nginx/error.log` contains lines like this (copied from similar issue #8967):

```
[error] 1006#1006: *9 open() "/var/www/nginx-default/50x.html" failed (2: No such file or directory), client: 110.138.116.120, server: meet.victorysolusi.com, request: "GET /b HTTP/1.1", upstream: "http://127.0.0.1:5000/b", host: "meet.victorysolusi.com"
```

**Analysis**

Whenever there is a problem with a backend service, nginx tries to render a 50x page. The `/etc/nginx/sites-available/bigbluebutton` configuration shipping with one of the bbb-packages contains the following lines:

```
        # Redirect server error pages to the static page /50x.html
        #
        error_page   500 502 503 504  /50x.html;
        location = /50x.html {
                root   /var/www/nginx-default;
        }
```

However, the `/var/www/nginx-default/50x.html` does not exist on a freshly installed BBB server. Because nginx is unable to find the error page, it will render a standard 404 instead of a 503 / 504, ...

Searching through the github issues for `404 nginx` turns up several issues where people are confused about why they are seeing 404 (when in fact backend services are down).

In order to resolve the issue, the lines cited above need to be removed from `/etc/nginx/sites-available/bigbluebutton`, or alternatively bigbluebutton needs to ship `/var/www/nginx-default/50x.html` as well.

**Comment by znerol** [comment]
Oh, so. I see that this has been fixed in bigbluebutton/bbb-install@4495f4b65782c7aada7c57ab16383b35937346bf 

```diff
@@ -868,7 +868,7 @@ server {
   #
   error_page   500 502 503 504  /50x.html;
   location = /50x.html {
-    root   /var/www/nginx-default;
+    root   /var/www/bigbluebutton-default;
   }
 }
 HERE
```

So, I guess that PR #13026 isn't actually doing the right thing?

**Comment by znerol** [comment]
However, I actually do not see `/var/www/bigbluebutton-default/50x.html` either on my BBB 2.3 installs. I'm confused.
